packages/opentelemetry-sdk-trace-base/src/export/ConsoleSpanExporter.ts

```diff
@@ -65,6 +65,7 @@ export class ConsoleSpanExporter implements SpanExporter {
       resource: {
         attributes: span.resource.attributes,
       },
+      instrumentationLibrary: span.instrumentationLibrary,
```

Comment:
InstrumentationLibrary is deprecated now in the code, should we use InstrumentationScope instead?